WooCommerce/Classes/Tools/UnitInputFormatter/CurrencySettings+Sanitized.swift

```diff
@@ -0,0 +1,15 @@
+import WooFoundation
+
+/// Sanitize separators that has spaces within it. Example ", " or ". "
+/// Separators with spaces are not supported by native iOS decimal formatters
+/// which causes the issue formatting and converting numbers to and from the decimals
+///
+extension CurrencySettings {
+    var sanitizedDecimalSeparator: String {
+        return decimalSeparator.replacingOccurrences(of: " ", with: "")
+    }
+
+    var sanitizedGroupingSeparator: String {
+        return groupingSeparator.replacingOccurrences(of: " ", with: "")
+    }
```

Comment:
```suggestion
```
This one is currently not used anywhere, so perhaps we want to remove it? If we don't, maybe we should rename it from `sanitizedGroupingSeparator` to `sanitizedThousandSeparator`?

The `CodingKey` in the `CurrencySettings` doesn't seem to follow the decoding pattern for this value: If we check the `Networking.SystemStatus+Settings` it's initially decoded as `case thousandSeparator = "thousand_separator"`, but at some point we started to name it "groupingSeparator" in CurrencySettings (perhaps because of the native NumberFormatter?) so perhaps makes sense to refactor the name here and there for consistency, wdyt?